webapp/.env.example

```diff
@@ -72,3 +72,8 @@ REACT_APP_MKR_MAX_CONVERSION_AMOUNT=
 REACT_APP_SNT_MAX_CONVERSION_AMOUNT=
 REACT_APP_ZIL_MAX_CONVERSION_AMOUNT=
 REACT_APP_RCN_MAX_CONVERSION_AMOUNT=
+
+# Redux
+
+REACT_APP_REDUX_ACTION_ONLY_TYPE=
+REACT_APP_REDUX_STORE_KEYS=
```

Comment:
Regardless of the comment above, maybe we should point out that these env variables are for the redux devtools extension **only**, to avoid confusion.

```
REACT_APP_DEVTOOLS_EXTENSION_ACTION_ONLY_TYPE
REACT_APP_DEVTOOLS_EXTENSION_STORE_KEYS
```

Reply:
Agree with adding `_DEVTOOLS_EXTENSION_`

But, how do you think that would be the use case for: 

```
REACT_APP_REDUX_ACTION_ONLY_DISPLAY_TYPE
REACT_APP_REDUX_ACTION_ONLY_SHOW_TYPE
REACT_APP_REDUX_ACTION_SKIP_PAYLOAD
```